scripts/monitoring/cron-send-security-updates-count.py

```diff
@@ -82,7 +82,7 @@ def get_security_updates(self):
             self.sec_updates = []
             raise subprocess.CalledProcessError(proc.returncode, subprocess.list2cmdline(cmd), output=stdout+stderr)
         lineslist = [line for line in stdout.split('\n') if line and 'Errno' not in line]
-        self.sec_updates = [UpdateItem(*re.split(r'\s+', line)) for line in lineslist]
+        self.sec_updates = [UpdateItem(*re.split(r'\s+', line)) for line in lineslist if re.split(r'\s+', line) == 3]
```

Comment:
This test for filtering away lines that didn't split neatly into 3 fields does mean we could be missing some important information (I know...I know...I recommended the easy fix to begin with :). Maybe it's better to break out this list comprehension into iterating through each line and catching/noticing when a line isn't formatted as expected.

@dbaker-rh thoughts?